mdata/cache/ccache_metric.go

```diff
@@ -12,16 +12,12 @@ import (
 type CCacheMetric struct {
 	sync.RWMutex
 
-	// points to the timestamp of the newest cache chunk currently held
-	// the head of the linked list that containes the cache chunks
-	newest uint32
-
-	// points to the timestamp of the oldest cache chunk currently held
-	// the tail of the linked list that containes the cache chunks
-	oldest uint32
-
 	// points at cached data chunks, indexed by their according time stamp
 	chunks map[uint32]*CCacheChunk
+
+	// instead of sorting the keys at each search we can save some ops by
+	// keeping the list until a chunk gets added or deleted
```

Comment:
this comment describes the change from previous code to new code, for which you should use the commit message. (which you already do). you should only mention here something like "maintain sorted list of chunk timestamps" or something which is more descriptive of the purpose of the variable.